### DISCUSSION TITLE
No location data

### DISCUSSION THREAD

**Discussion by Defender110SE** [tesla-android/issue-tracker#247] [Q&A] [answered]
Hi!
I got a Raspberry 4, 4GB RAM with 2023.7.1 and a Mikrotik LtAP providing the internet connection via cable. 
All seems to work, except there is no location data available to any of the apps installed (Maps, ABRP,...).
My Model Y is on software 2023.6.8.
To my understanding the Tesla browser should supply this information to the Raspberry?
What did I miss?
Thank you!

**Comment by Defender110SE** [comment]
Doesn't anyone have any idea? Is this to be expected behavior or did I miss something?

**Reply by mikegapinski** [reply, marked as answer]
this feature is not available yet, that is why it's not mentioned anywhere

**Reply by Defender110SE** [reply]
Thanks a lot for the feedback and great work! Clear! Will keep my eyes open for updates.

**Comment by yerode** [comment]
I suspect location data isn't supposed to be activated or to work in the v2023.18.1 yet? Aside from that. Great build.

**Reply by Defender110SE** [reply]
Yes. Thats what Mike confirmed. I was just not aware of this and suspected I did something wrong.
Kind of a pity, since this is the most important function for me to use navigation outside of Teslas service area (currently based in the Republic of Georgia).
But I guess with all the work thats put into this project, its just a matter of time.

**Reply by yerode** [reply]
Thanks. Yes, it's a pity. For me it's also the most important feature. I won't use Carplay/AndroidAuto and look forward to simply use ABRP and GMaps directly on TeslaAndroid.

**Reply by yerode** [reply]
I suspect there won't be working location data in the 2023.20.1?
